BlocSpot/app/src/main/java/com/izor066/android/blocspot/ui/adapter/ItemAdapter.java

```diff
@@ -0,0 +1,95 @@
+package com.izor066.android.blocspot.ui.adapter;
+
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.TextView;
+
+import com.izor066.android.blocspot.BlocSpotApplication;
+import com.izor066.android.blocspot.R;
+import com.izor066.android.blocspot.api.DataSource;
+import com.izor066.android.blocspot.api.model.PointOfInterest;
+
+/**
+ * Created by igor on 28/9/15.
+ */
+public class ItemAdapter extends RecyclerView.Adapter<ItemAdapter.ItemAdapterViewHolder> {
+
+    public static interface OnPointOfInterestClickListener {
+        public void OnPointOfInterestClick(PointOfInterest pointOfInterest);
```

Comment:
Method names should be lower case i.e. `onPointOfInterestClick(PointOfInterest pointOfInterest)`